tests/integration/test_parallel_resource_executor_processor.py

```
@@ -0,0 +1,741 @@
+        from file_organizer.parallel.config import ParallelConfig
+        from file_organizer.parallel.processor import ParallelProcessor
+
+        files = []
+        for i in range(4):
+            f = tmp_path / f"fps{i}.txt"
+            f.write_text("data", encoding="utf-8")
+            files.append(f)
+
+        cfg = ParallelConfig(max_workers=2, retry_count=0, timeout_per_file=30.0)
+        proc = ParallelProcessor(cfg)
+        result = proc.process_batch(files, lambda p: "ok")
```

Ruff (0.15.9)

[warning] Unused lambda argument: `p`

(ARG005)